features/send_email.feature

```diff
@@ -27,19 +27,34 @@ Feature: Test if user is able to send message.
     Then I click the 'View' link
     And I should see 'Heia Northug' on my screen
     And I type in 'Heja Carlson' in field for 'message_body'
-    And I click the 'Reply' button
+    Then I click the 'Reply' button
     And I should see 'Your reply message was successfully sent!' on my screen
 
   @javascript
   Scenario: Holger recieves a message from Kalle and deletes it
     Given Kalle have sent an message to Holger
     And I am logged in as 'Holger'
     And I am on the inbox page
-    Then I should see 'Heia Norge' on my screen
+    And I should see 'Heia Norge' on my screen
     Then I click the 'View' link
     And I should see 'Heia Northug' on my screen
-    And I click the 'Move to trash' link
-    And I accept the alert
+    Then I click the 'Move to trash' link
+    Then I accept the alert
     Then I click the 'Trash' link
     And I should see 'Heia Northug' on my screen
 
+  @javascript
+  Scenario: Holger recieves a message from Kalle and deletes it and then untrash it.
+    Given Kalle have sent an message to Holger
+    And I am logged in as 'Holger'
+    And I am on the inbox page
+    And I should see 'Heia Norge' on my screen
+    Then I click the 'View' link
+    And I should see 'Heia Northug' on my screen
+    Then I click the 'Move to trash' link
+    And I accept the alert
+    Then I click the 'Trash' link
+    And I should see 'Heia Northug' on my screen
+    Then I click the 'View' link
+    Then I click the 'Untrash' link
+    And I should see 'Heia Northug' on my screen
```

Comment:
Three scenarios were kalle has sent message to holger, this can be extracted to the background to keep this dry.